docs/apis/subsystems/tag/index.md

```diff
@@ -0,0 +1,274 @@
+---
+title: Tag API
+tags:
+- API
+- Sub System
+- Tag
+- Tags
+---
+
+## Tag API overview
+
+The Tag API allows you to assign labels to information in Moodle. This makes finding this information easier and also facilitates the grouping of similar information. The Tag API allows you to create, modify, delete and search tags in the Moodle system. The main tag related functions can be found in the tag/classes/tag.php file. For a thorough overview of all of the functions available for working with Tags please see methods in core_tag_tag, core_tag_collection and core_tag_area classes, however, the following examples should give you a general understanding of how to get started with tags.
+
+This page describes API in Moodle 3.1 and above, for earlier versions see [Tag API before 3.1](https://docs.moodle.org/dev/Tag_API_before_3.1)
+
+## Tag API usage
+
+When a user tags something a **tag instance** is created in the database linking the item to the actual **tag**. If the tag did not exist before it is created automatically. Do not confuse these two entities - deleting the tag instance does not normally delete tag, however deleting tag deletes all tag instances associated with it.
+
+Developers define **tag areas** the areas that can be tagged, examples are:
+
+- blog posts
+- courses
+- users (tags represent user interests)
+- activity modules
+- questions
+- wiki pages
+
+Each tag area is identified by two attributes - component and itemtype. *Itemtype must be a name of a table in the database.* Component is the core component or plugin responsible for the tagging. This way the same DB table (for example 'user' or 'course') may be independently tagged by several components. Administrator or manager is able to manage the tag areas, collections and tags inside them on the [Managing tags](https://docs.moodle.org/en/Managing_tags) page. Users are able to search tags and view all items tagged with them that they have access to.
+
+### Defining a tag area
+
+First, developer must define the tag areas in the file **db/tag.php**. This will usually look like:
+
+```php
+$tagareas = array(
+    array(
+        'itemtype' => 'wiki_pages',  // This must be a name of the database table (without prefix).
+        'component' => 'mod_wiki', // This can be omitted for plugins since it can only be the full frankenstyle name of the plugin.
+        'callback' => 'mod_wiki_get_tagged_pages',
+        'callbackfile' => '/mod/wiki/locallib.php',
+    ),
+);
+```
+
+You will also need to add language string, for the example above it will be **$string[]('tagarea_wiki_pages') = 'Wiki pages';**
```

Comment:
This is the line causing the build failure.

In the wikitext, this was just bolded, and unfortunately our regexes to convert interwiki links aren't quite perfect so have tried to convert it to a link.

Obviosuly, it should be:
```suggestion
You will also need to add language string, for the example above it will be `$string['tagarea_wiki_pages'] = 'Wiki pages';`
```